api/zeebe/zeebe-openapi.yaml

```diff
@@ -298,6 +322,8 @@ components:
       type: object
       properties:
         changeset:
+          allOf:
+            - $ref: "#/components/schemas/Changeset"
```

Comment:
this is the change that prompts the generator to select the wrong `description`.

Reply:
Can we, for the time being, simply use the expected description in the `Changeset` schema instead of `A map of changes`? That way, the result is correct for now. When the issue in #3568 is fixed, we can use the OpenAPI YAML as is. Wdyt?

Reply:
If we do this, we should probably add a note to the OpenAPI file in the docs as well so we're aware the next time we adjust it (given we do that before the issue is fixed).

Reply:
This is a good idea! I'll add a step to the pre-generation script to update the description of `Changeset`, and mark the step clearly as "temporary until the generator picks the correct description." Commit coming soon, on this PR.

Reply:
@tmetzke take a look at https://github.com/camunda/camunda-docs/pull/3567/commits/6d3d3725e6b26b1a6cf3bf934f66e195af126279. It hacks the description of the Changeset schema object, using a hardcoded value.

I started down a rabbit hole of (1) trying to capture the description from the UserTaskUpdateRequest.changeset property, (2) assigning it to the description of the Changeset, then (3) correcting the formatting, but it got pretty deep and I didn't think it was worth it. The risk of this current solution is that if the source description changes, we'll need to update this code.

Since it's a hack anyway, I thought that was appropriate, instead of spending the next day trying to figure out how to either customize the output of https://github.com/nodeca/js-yaml so that it doesn't create 300 unimportant formatting differences, or capturing _just_ one property from output re-generated by js-yaml and putting it in the correct place.

Reply:
I love that you tried to make this as generic as possible, I was also wondering if that is easily achievable 👍
Since it looks like it's not, let's go with a pragmatic approach and fix it manually, plus, document it there as well.